.github/workflows/make-check.yml

```diff
@@ -26,3 +26,6 @@ jobs:
           go-version: ${{ matrix.go }}
       - name: Build
         run: make check
+      - if: runner.os == 'Linux'
```

Comment:
Is this problematic to run this on macos? Would be simpler to avoid conditional steps (our github workflows deserve a big rethinking/rework anyway into smaller/reusable workflows, some which run on all platforms, some whihc only run on one platform)

Reply:
Always useful to have these details in the commit log...